### PR TITLE
Carlos

### PR DESCRIPTION
**Pull request by ccespedes**

### CODE REVIEW

**Review by belunatic — APPROVED** [review]

File: expense-splitter/src/components/context/SiteContext.jsx [hunk 1]
```diff
@@ -1,60 +1,42 @@
...
-        {
-          name: 'Urth Cafe Brunch',
-          description: 'Brunch with the baseball team',
-          budget: '120',
-          id: 'P_PNnrbjurvmasxRtXEov',
-        },
-      ]
+export const DataProvider = ({ children }) => {
+  // initialize data from localStorageDB
+  const data = db
```
Comment (singhalex) [inline expense-splitter/src/components/context/SiteContext.jsx]: This data declaration is unnecessary. Simply importing db from the utility file will initialize it.

File: expense-splitter/src/components/context/SiteContext.jsx [hunk 2]
```diff
@@ -1,60 +1,42 @@
-import React, {useState} from 'react'
+import React, { useState } from 'react'
```
Comment (singhalex) [inline expense-splitter/src/components/context/SiteContext.jsx]: Importing React is no longer needed

**Review by singhalex — CHANGES_REQUESTED** [review]
Looks good so far. Just some minor edits before approval

**Comment by singhalex** [comment]
Did you intend to implement saving to localStorage in this PR or will that be implemented in a future update?

**Review by kristi-h — APPROVED** [review]

File: expense-splitter/src/components/context/SiteContext.jsx (hunk 1 above)
Comment (ccespedes) [reply expense-splitter/src/components/context/SiteContext.jsx]: That was just there for console logging. It's removed.

**Review by singhalex — APPROVED** [review]
Looks good!

**Pull request merged**